packages/moonstone/DropDown/tests/DropDown-specs.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import {DropDown, DropDownBase} from '../DropDown';
+
+const title = 'Dropdown select';
+const children = ['option1', 'option2', 'option3'];
+
+describe('DropDownBase', () => {
```

Comment:
Just curious why you're testing the base component separately instead of running them all on the stateful one?